### PR TITLE
[v7r0]  Fix cleaning empty directory, revive DataManager tests

### LINKED ISSUE #4420: DMS - Error deleting empty directory
I have an error when I delete an empty directory on the DFC.
```
$ dmkdir test_to_delete
$ drm -r test_to_delete
ERROR: FileCatalog error ( 1604 : Failed to perform isDirectory from any catalog)

0 object(s) removed in total
1 object(s) failed removal in total
```

It works if it isn't empty

```
$ dput test_to_delete plop.txt
$ drm -r test_to_delete

1 object(s) removed in total
```

I have the same behavior and error with `dirac-dms-clean-directory`. But it works with `dirac-dms-filecatalog-cli`.

```
$ dmkdir test_to_delete
$ dirac-dms-filecatalog-cli -f BiomedFileCatalog
Starting BiomedFileCatalog client

File Catalog Client $Revision: 1.17 $Date: 
            
FC:/> cd /[...]/axel_bonnet
FC:/[...]/[...]/axel_bonnet>rmdir test_to_delete
lfn: /[...]/axel_bonnet/test_to_delete
Directory /[...]/axel_bonnet/test_to_delete removed from the catalog
```
I feel this is linked to a former issue I made concerning directory deletion #4012 . This issue have been corrected in v6r21p14.
I just had a test with an previous release (v6r21p1) and the sequence `$ dmkdir test_to_delete; drm -r test_to_delete` worked then.

### PR DESCRIPTION
**Pull request by andresailer**
Currently trying to revive the DataManager tests, installing two local SEs and copying files. Adding some tests for cleaning directories once this is working.

BEGINRELEASENOTES

*DMS
FIX: dirac-dms-clean-directory: also works on single empty directory, fixes #4420 

*Framework
FIX: dirac-install-component: fix bug prohibiting use of the -m/--module parameter

ENDRELEASENOTES

### CODE REVIEW

**Comment by andresailer** [comment]
I think my new test_cleanDirectory is failing because the integration test is not actually using my feature branch to run the client tests? At least that is what I saw also when running the tests locally.
Can someone confirm my suspicion?

File: tests/Integration/DataManagementSystem/Test_DataManager.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
+from __future__ import print_function
```
Comment (fstagni) [inline tests/Integration/DataManagementSystem/Test_DataManager.py]: Please add a few lines of docstring, also writing what's needed for this test to be run

**Comment by fstagni** [comment]
Interesting. From the output, the command used for installing the server is
```
dirac-install.py -ddd --dirac-os --dirac-os-version=master --module=/home/dirac/LocalRepo/ALTERNATIVE_MODULES/repo:::DIRAC:::local /home/dirac/ServerInstallDIR/install.cfg /home/dirac/ServerInstallDIR/dirac-ci-install.cfg
```
and those used for the client is:
```
dirac-install -r v7r0p11 -t client --module=ALTERNATIVE_MODULES --dirac-os --dirac-os-version=master /home/dirac/ClientInstallDIR/dirac-ci-install.cfg -ddd
```
I don't have time to check properly now. @chrisburr does it ring a bell to you?

**Comment by chrisburr** [comment]
Yes, sorry about that, it's a bug that came from #4409 I've already fixed in #4429. In `tests/Jenkins/utilities.sh` this change should be made:
```diff
-  export CLIENT_ALTERNATIVE_MODULES=${CLIENT_ALTERNATIVE_MODULES:-ALTERNATIVE_MODULES}
+  export CLIENT_ALTERNATIVE_MODULES=${CLIENT_ALTERNATIVE_MODULES:-${ALTERNATIVE_MODULES}}
```
Though that way of fixing it will conflict with #4429.

**Comment by andresailer** [comment]
If it is just one line, maybe you can make a new PR fixing that problem, and everyone rebases on top of that?

File: tests/Integration/DataManagementSystem/Test_DataManager.py (hunk 1 above)
Comment (andresailer) [reply tests/Integration/DataManagementSystem/Test_DataManager.py]: Added docstring and requirements

File: tests/Jenkins/utilities.sh [hunk 2]
```diff
@@ -737,6 +747,29 @@ diracServices(){
...
+diracSEs(){
+  echo '==> [diracSEs]'
+
+  echo "==> Installing SE-1"
+  seDir=$SERVERINSTALLDIR/Storage/SE-1
+  mkdir -p $seDir
+  if ! dirac-install-component DataManagement SE-1 -m StorageElement -p BasePath=$seDir -p Port=9148 "$DEBUG"; then
+      echo 'ERROR: dirac-install-component failed'
+      exit 1
+    fi
```
Comment (chrisburr) [inline tests/Jenkins/utilities.sh]:
```suggestion
  fi
```

File: tests/Jenkins/utilities.sh [hunk 3]
```diff
@@ -737,6 +747,29 @@ diracServices(){
...
+
+  echo "==> Installing SE-2"
+  seDir=$SERVERINSTALLDIR/Storage/SE-2
+  mkdir -p $seDir
+  if ! dirac-install-component DataManagement SE-2 -m StorageElement -p BasePath=$seDir -p Port=9147 "$DEBUG"; then
+      echo 'ERROR: dirac-install-component failed'
+      exit 1
+    fi
```
Comment (chrisburr) [inline tests/Jenkins/utilities.sh]:
```suggestion
  fi
```

**Comment by andresailer** [comment]
Fixed the indentation in the shell file

**Pull request merged**